content/en/docs/languages/go/instrumentation.md

```diff
@@ -169,18 +169,39 @@ are useful for aggregating, filtering, and grouping traces. Attributes can be
 added at span creation, or at any other time during the lifecycle of a span
 before it has completed.
 
+Generally speaking, you should try to set attributes during span creation.
+This will incur less overhead compared to setting attributes after
+span creation.
+
+The following example adds attributes at creation:
+
 ```go
 // setting attributes at creation...
-ctx, span = tracer.Start(ctx, "attributesAtCreation", trace.WithAttributes(attribute.String("hello", "world")))
-// ... and after creation
-span.SetAttributes(attribute.Bool("isTrue", true), attribute.String("stringAttr", "hi!"))
+ctx, span = tracer.Start(ctx, "attributesAtCreation",
+	trace.WithAttributes(attribute.String("hello", "world"))
```

Comment:
Have you checked if it works? AFAIK it does not. See https://github.com/open-telemetry/opentelemetry-go/issues/3368

Reply:
This works for me on the getting started guide:

```
	ctx, span := tracer.Start(
		r.Context(),
		"roll",
		trace.WithAttributes(attribute.String("game", "craps")),
	)
```

When run:

```
        "Attributes": [
                {
                        "Key": "game",
                        "Value": {
                                "Type": "STRING",
                                "Value": "craps"
                        }
                },
```

Reply:
You are right. My bad. The issue was about getting a tracer/meter with scope attributes and this creates a span.